### PR TITLE
KAFKA-14662: Update the ACL list in the doc

### PR DESCRIPTION
**Pull request by tinaselenge**
Add the missing ACLs to the doc. 

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: docs/security.html [hunk 1]
```diff
@@ -2089,6 +2089,144 @@ <h5 class="anchor-heading"><a id="operations_resources_and_protocols" class="anc
...
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DESCRIBE_QUORUM (55)</td>
+            <td>Alter</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
```
Comment (showuon) [inline docs/security.html]: Duplicated entry. And it should be `Describe` `Cluster`, right?

File: docs/security.html [hunk 2]
```diff
@@ -2089,6 +2089,144 @@ <h5 class="anchor-heading"><a id="operations_resources_and_protocols" class="anc
...
+            <td>ClusterAction</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>UPDATE_FEATURES (57)</td>
+            <td>Alter</td>
+            <td>Cluster</td>
+            <td></td>
+        </tr>
```
Comment (showuon) [inline docs/security.html]: Duplicated entry.

**Review by showuon — COMMENTED** [review]
@tinaselenge , nice find! Left some comments.

File: docs/security.html (hunk 2 above)
Comment (tinaselenge) [reply docs/security.html]: Thank you @showuon. Looking at the other ApiKeys, they are listed several times for each operation primitive and resource it checks, e.g FETCH. I followed the same format for DESCRIBE_QUORUM and UPDATE_FEATURES as they both had clusterAction set to true here (https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java#L101).  However now looking at the code again, I don't think ClusterAction operation is checked as part of the authorization check for them but used later for throttling the requests. So I think I should remove them, what do you think?

File: docs/security.html (hunk 1 above)
Comment (tinaselenge) [reply docs/security.html]: Re duplication, I commented below. And yes, it should have been Describe. Thanks for spotting that :)

File: docs/security.html (hunk 2 above)
Comment (showuon) [reply docs/security.html]: I think we should remove them since the permission is not required for this API key.

**Review by showuon — APPROVED** [review]
LGTM! Thanks for the PR.

**Comment by showuon** [comment]
Retriggering the CI build since it failed with infra's issue.
https://ci-builds.apache.org/job/Kafka/job/kafka-pr/job/PR-13660/3/

**Comment by showuon** [comment]
Failed tests are unrelated:
```
    Build / JDK 8 and Scala 2.12 / integration.kafka.server.FetchFromFollowerIntegrationTest.testRackAwareRangeAssignor()
    Build / JDK 11 and Scala 2.13 / org.apache.kafka.tools.MetadataQuorumCommandTest.[1] Type=Raft-Combined, Name=testDescribeQuorumReplicationSuccessful, MetadataVersion=3.5-IV2, Security=PLAINTEXT
    Build / JDK 11 and Scala 2.13 / org.apache.kafka.tools.MetadataQuorumCommandTest.[2] Type=Raft-Isolated, Name=testDescribeQuorumReplicationSuccessful, MetadataVersion=3.5-IV2, Security=PLAINTEXT
```

**Comment by showuon** [comment]
backported to v3.5 branch.

**Pull request merged**